build_scripts/build_usd.py

```diff
@@ -762,6 +758,10 @@ def InstallBoost_Helper(context, force, buildArgs):
                                 " linkflags=\"{0}\"".format(macOSArch)
             bootstrapCmd += " --with-toolset=clang"
 
+            # If Python2 is also installed in the system then it is sometimes
+            # found first, so we force the bootstrap to Python3
+            bootstrapCmd += " --with-python-version=3"
+
```

Comment:
It seems like this ought to only be needed if `context.buildPython` is enabled. I think it could also use the information returned by `GetPythonInfo` to specify the full version, like:

```
if context.buildPython:
    pythonInfo = GetPythonInfo(context)
    bootstrapCmd += " --with-python-version={ver}".format(ver=pythonInfo[3])
```

In theory this seems like this should be independent of macOS but I'm OK with leaving it in this block for now and just adding a comment saying something to that effect.

Reply:
After testing this on the latest public release of macOS Ventura it seems that the `--with-python-version=3` option isn't necessary.  It only causes a problem for our internal builds.  So I think I can remove it.  The other change, to remove the `combined` option is needed with 1.78